src/components/Map/GoogleMapComp.tsx

```diff
@@ -1,30 +1,66 @@
-import getCoordsFromAddress, { Coord, House } from '@/utils/services/getCoordsFromAddress';
-import { useEffect, useState } from 'react';
-import { GoogleMap, Marker, withGoogleMap, withScriptjs } from 'react-google-maps';
+import { House } from '@/utils/services/getCoordsFromAddress';
+import { useState } from 'react';
+import { GoogleMap, Marker, withGoogleMap, withScriptjs, InfoWindow } from 'react-google-maps';
 import styles from './GoogleMapComponent.module.scss';
 import CSS from 'csstype';
 
+type Houses = {
+  _id: number;
+  city: string;
+  street: string;
+  descriptionField: string;
+  lat: number;
+  lng: number;
+};
 
-type HouseDataAndStyleCSS = {houses:House, style:CSS.Properties}
-export default function GoogleMapComp({ houses, style }:HouseDataAndStyleCSS) {
+type SetHousesCoord = React.Dispatch<React.SetStateAction<Houses[]>>;
+
+async function fetchAndSetHousesCoords(setHousesCoords: SetHousesCoord) {
+  const res = await fetch('https://pacific-refuge-80597.herokuapp.com/api/houses');
```

Comment:
There's the .env variable `VITE_BACKEND_URL` that could replace the part `https://pacific-refuge-80597.herokuapp.com/api`.